scripts/docker-build-lib-linux.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
This file has been renamed from or recreated in place of `docker-manylinux.sh`, but it's essentially that. We could rename it back.